design-library/src/tokens/tailwind/transitions.ts

```diff
@@ -0,0 +1,10 @@
+export const keyframes = {
+	wiggle: {
```

Comment:
These should probably be prefixed with `bcc` to prevent clashes with local applications.

Reply:
Hmm, we don't prefix our other tailwind variables? Idea is that this is the one and only wiggle :D

Reply:
True, but maybe we should have? I guess the chance of a conflict is fairly low in this case, but it feels like a responsible thing to do as a library author 😄.